requirements/lint.txt

```diff
@@ -314,14 +313,9 @@ zope-schema==6.2.0 \
     # via mypy-zope
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==22.2 \
```

Comment:
Any idea why this is being removed?

Reply:
Yes, types-s3transfer dropped it in https://github.com/youtype/types-s3transfer/commit/16b7b3a07e76aa4acfdb035967143f03293c272f